### ISSUE TITLE
implement noreset logic for TGFX16-CD

### ISSUE THREAD

**Issue by tjanas** [MiSTer-devel/Main_MiSTer#535]
Similar to the functionality implemented by https://github.com/MiSTer-devel/Main_MiSTer/commit/7d896cd85a2c5723b9c5ad43c48275cb93ba52dc , there is one PC Engine CD [game](http://www.pcengine.co.uk/HTML_Games/CD_Battle-Heroes_of_Light.htm) that prompts the player to insert a red book audio CD. Could the noreset logic used for PSX be applied to support this game functionality?

**Comment by thehughhefner** [comment]
@dshadoff

**Comment by dshadoff** [comment]
That game is terrible.  It is also the only game that I am aware of which reads redbook audio to the system as data.
Without saying "no", this is a very low "effort to reward" ratio.
I would say that CD+G support is a far higher priority.

**Comment by tjanas** [comment]
I fully agree that the game is terrible, that it should be a very low-priority item, and that CD+G is a higher priority; nonetheless I think it should be implemented at some point in the future for completeness.

**Comment by dshadoff** [comment]
Just wanted to be clear that you understand priority is low.
